ios/Makefile

```diff
@@ -12,7 +12,7 @@ test:
 	xcodebuild clean test \
 		-project $(project_path) \
 		-scheme $(scheme_name) \
-		-destination "platform=iOS Simulator,name=iPhone 11,OS=13.3"
+		-destination "platform=iOS Simulator,name=iPhone 11,OS=14.4"
```

Comment:
That's not quite ideal, since iOS 13.3 is our lowest supported version. Would be good to see if we can't get the older emulator back on GitHub's workers... But it's still better than a failing build if we test only 14.4.

Reply:
The problem is that there does not seem to be any other OS available. These are the platforms:

```
	{ platform:iOS Simulator, id:B7C37619-BA36-424D-B938-132F83363FCD, OS:14.4, name:iPad (7th generation) }
		{ platform:iOS Simulator, id:AFBB8BA9-2CE0-4D7F-999B-770713FD10EF, OS:14.4, name:iPad (8th generation) }
		{ platform:iOS Simulator, id:9EB83B11-C00F-4F8A-925C-BA3619BB931B, OS:14.4, name:iPad Air (3rd generation) }
		{ platform:iOS Simulator, id:9CD40043-30E1-4E28-B8F5-EF198D55D3A2, OS:14.4, name:iPad Air (4th generation) }
		{ platform:iOS Simulator, id:75815B2E-87B9-4BD8-AE35-915EA4112D36, OS:14.4, name:iPad Pro (9.7-inch) }
		{ platform:iOS Simulator, id:1812C170-B3E3-406B-BD19-D759B8916FD5, OS:14.4, name:iPad Pro (11-inch) (2nd generation) }
		{ platform:iOS Simulator, id:9D4B1AEE-8EE8-49DE-9BEE-1A5E7D4E3507, OS:14.4, name:iPad Pro (12.9-inch) (4th generation) }
		{ platform:iOS Simulator, id:2FD5C51E-131F-42EF-8F68-8DCF6389144E, OS:14.4, name:iPhone 8 }
		{ platform:iOS Simulator, id:D6B9EF96-4E31-415E-8990-F2AD3F0BDAB5, OS:14.4, name:iPhone 8 Plus }
		{ platform:iOS Simulator, id:EAD2AC5A-D0FE-41C0-8F70-FBBC7460CF69, OS:14.4, name:iPhone 11 }
		{ platform:iOS Simulator, id:003786C0-2A5E-412B-A9F1-10A664FD93EC, OS:14.4, name:iPhone 11 Pro }
		{ platform:iOS Simulator, id:C78245F3-BC29-4E8B-84D1-EA1D69893C7B, OS:14.4, name:iPhone 11 Pro Max }
		{ platform:iOS Simulator, id:B483A12A-3A4D-44A4-949E-5E5B523B8A1C, OS:14.4, name:iPhone 12 }
		{ platform:iOS Simulator, id:B4072E75-75F3-459C-92AB-A3ECD5F4CDAA, OS:14.4, name:iPhone 12 Pro }
		{ platform:iOS Simulator, id:3163E6F7-9CFA-45B7-AA7D-C51484A8D5BE, OS:14.4, name:iPhone 12 Pro Max }
		{ platform:iOS Simulator, id:156CDD5C-0B1F-4E5F-A8CA-42D5B98FB663, OS:14.4, name:iPhone 12 mini }
		{ platform:iOS Simulator, id:7AE89782-2796-4CF4-9020-E40E3B43E52F, OS:14.4, name:iPhone SE (2nd generation) }
		{ platform:iOS Simulator, id:10E0C3AF-82E2-4623-84C1-4F59B67CEC39, OS:14.4, name:iPod touch (7th generation) }
```
If you want, I can try and see what's the origin of those destinations before merging. It might simply be that it's implicit with the macos-latest runner. We might want to create a matrix to run other versions.

Reply:
[This](https://github.com/actions/virtual-environments/issues/551) is what came up with. Apparently just one version is installed by default, so we can either try that or else check what versions are installed in other macos runners.

Reply:
OK, I'm missing something here. Theoretically "latest" runner should be 10.15, and that should support 13.3. The Catalina runner, which should not be an alias of latest, only supports 13.7 and 14.4. So I might want to try to set multiple OS here, with different OS versions. Shouldn't be hard to do with Makefiles. We can also leave that for later if you want.

Reply:
Ideally we'd run the tests on multiple versions, but if we can somehow influence what runner we get (so that it has `13.3`), that'd be good enough for now I'd say! If we _can_ have multiple runners, to be economical, I would suggest we go for the _lowest supported version_ (`13.3`) and the _latest_ (`15.0`?)

Reply:
I'll see what I can do. Apparently, [15.0 is supported](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md#installed-sdks)

Reply:
And right, [latest started using macOS-11 last month](https://github.com/actions/virtual-environments/issues/4060), and it hadn't yet bubbled up to the documentation. Let me run a couple of tests to see if what you say above can be done.

Reply:
Hum, no, it's using 10.15 all the time.